Warmup-1/startOz/test/index.test.ts

```diff
@@ -0,0 +1,23 @@
+// Given a string, return a string made of the first 2 chars (if present), however include first char only if it is 'o' and include the second only if it is 'z', so "ozymandias" yields "oz".
+
+import { assert } from "chai";
+import { startOz } from "../src";
+
+// startOz("ozymandias") → "oz"
+// startOz("bzoo") → "z"
+// startOz("oxx") → "o"
+
+describe("startOz", () => {
```

Comment:
Please add a test for when there is no `o` or `z` in the string

Reply:
Done.